features/kernel/src/main/feature/feature.xml

```diff
@@ -17,8 +17,6 @@
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
     <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
-    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
-    <repository>mvn:io.fabric8/fabric8-karaf-features/${fabric8.version}/xml/features</repository>
```

Comment:
✏️ I don't see `fabric8.version` being used anywhere else. Can that property be removed in the root pom?

Reply:
Yep, I removed it. Thanks